js/exporter/jspdf/common/rows_splitting.js

```diff
@@ -126,7 +179,7 @@ function splitRectsByPages(rects, marginValue, coordinate, dimension, checkPredi
                 return false;
             }
         });
-
+        const { currentPageRectsContainsOnlyHeader, multiPageRowPages } = getMultiPageRowPages(currentPageRects, rectsToSplit, pages.length === 0, onSplitMultiPageRow, isFitToPageForMultiPageRow);
```

Comment:
After splitting ```getMultiPageRowPages``` let's do some changes here too:

```suggestion
        const isCurrentPageContainsOnlyHeader = checkPageContainsOnlyHeader(currentPageRects, pages.length === 0);
        const multiPageRowPages = getMultiPageRowPages(
          currentPageRects,
          rectsToSplit,
          isCurrentPageContainsOnlyHeader,
          onSplitMultiPageRow, 
          decorateIsFitToPageFunc(isFitToPage, marginValue),
        );
```